docs/user/polarization.rst

```diff
@@ -348,39 +348,63 @@ streams.
 Process Variables
 -----------------
 
-Process variables (PV) indicate the spin selector used and its state. Given a
-particular Polarizer, PV PolarizerFlipper indicates whether the selected state is "up" or "down".
-The PV AnalyzerFlipper plays an analogous role for the analyzer.
-The following tables summarize the possible values of the polarization PVs.
-
-+-----------------------+---------------------+
-| PV Polarizer          | PV PolarizerFlipper |
-+=======================+=====================+
-| 0 - No Polarizer      |                     |
-+-----------------------+---------------------+
-| 1 - Reflection        | 1 - ON              |
-+-----------------------+---------------------+
-| 1 - Reflection        | 0 - OFF             |
-+-----------------------+---------------------+
-| 2 - Transmission      | 1 - ON              |
-+-----------------------+---------------------+
-| 2 - Transmission      | 0 - OFF             |
-+-----------------------+---------------------+
-| 3 - Undefined         |                     |
-+-----------------------+---------------------+
-
-+-----------------------+--------------------+
-| PV Analyzer           | PV AnalyzerFlipper |
-+=======================+====================+
-| 0 - No Analyzer       |                    |
-+-----------------------+--------------------+
-| 1 - Fan & SF2         | 1 - ON             |
-+-----------------------+--------------------+
-| 1 - Fan & SF2         | 0 - OFF            |
-+-----------------------+--------------------+
-| 2 - 3He               | 1 - ON             |
-+-----------------------+--------------------+
-| 2 - 3He               | 0 - OFF            |
-+-----------------------+--------------------+
-| 3 - Undefined         |                    |
-+-----------------------+--------------------+
+Process variables (PV) indicate the spin selectors used and their state.
+
+- **Polarizer**: (int single-value) type of device to polarize the upstream neutron beam
+- **PolarizerFlipper**: (int time-series) the device placed between the Polarizer and the sample can flip the Sz state.
```

Comment:
What does "Sz" stand for? Does it just mean spin?

Reply:
It's the component of the spin along the polarizing direction, usually taken to be the Z-axis. I'll explain it in the docs